bundles/product-list-search-rest-api-extension/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "fond-of-oryx/product-list-search-rest-api-extension",
+  "description": "ProductListSearchRestApiExtension bundle.",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "József Geng",
+      "email": "gengjozsef86@gmail.com"
+    }
+  ],
+  "require": {
+    "php": ">=7.3"
+  },
+  "require-dev": {
+    "fond-of-codeception/spryker": "^1.0",
+    "spryker/code-sniffer": "^0.12",
+    "spryker-sdk/phpstan-spryker": "*"
+  },
+  "minimum-stability": "dev",
+  "prefer-stable": true,
+  "autoload": {
+    "psr-4": {
+      "FondOfOryx\\": "src/FondOfOryx/"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "FondOfOryx\\": "tests/FondOfOryx/",
+      "Generated\\": "src/Generated/"
+    }
```

Comment:
plugin config for phpcs is missing

Reply:
@julianzimmermann  thanx for the remark. I've resolved it.